sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -694,6 +694,7 @@ object FunctionRegistry {
     expression[MapEntries]("map_entries"),
     expression[MapFromEntries]("map_from_entries"),
     expression[MapConcat]("map_concat"),
+    expression[SortMap]("sort_map"),
```

Comment:
We should also have this in functions.scala and functions.py. Should better file JIRAs, and add TODOs so we don't lose the track